webapp/channels/src/sass/responsive/_mobile.scss

```diff
@@ -167,7 +167,7 @@
                 }
 
                 .nav-pills {
-                    >li {
+                    > &__tab {
```

Comment:
@ayush-chauhan233 @Kshitij-Katiyar I know this is an old PR but it has introduced a CSS syntax error that is throwing warnings during build.  Apparently this will throw an actual error when we update the build tools.

<img width="1041" alt="Screenshot 2025-05-12 at 3 39 59 PM (1)" src="https://github.com/user-attachments/assets/dc3a40ba-5b7a-4d8a-ae1f-9419d0d6945d" />

Reply:
I'll submit a PR to remove it. I don't think this CSS actually has any effect because the only thing inside of this is an `a` element which doesn't exist inside of any `.nav-pills__tab` in the app from what I can tell